apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -546,6 +552,18 @@ exported_definitions(Module, POIKind, ExportFormat) ->
       []
   end.
 
+-spec module_name(els_dt_document:item()) -> [binary()].
+module_name(#{uri := Uri}) ->
+  ModuleName = filename:basename(lists:last(filename:split(Uri)),
```

Comment:
The `els_uri` module offers a conversion function from _Uri_ to _Module_. Since in this case we don't need the conversion to atom part, I would recommend modifying that function with something like (untested):

```
module(Uri) ->
  module(Uri, to_atom).

module(Uri, to_atom) ->
  binary_to_atom(module(Uri, to_binary), utf8);
module(Uri, to_binary) ->
  filename:basename(path(Uri), <<".erl">>).
```